packages/mantine/src/theme/Plasmantine.tsx

```diff
@@ -1,10 +1,12 @@
 import {MantineProvider} from '@mantine/core';
+import {Notifications} from '@mantine/notifications';
 import {FunctionComponent, PropsWithChildren} from 'react';
 
 import {plasmaTheme} from './Theme';
 
 export const Plasmantine: FunctionComponent<PropsWithChildren> = ({children}) => (
     <MantineProvider withGlobalStyles withNormalizeCSS theme={plasmaTheme}>
+        <Notifications position="top-center" />
```

Comment:
Not sure we really want to sneak in the notifications in there. It is not obvious to guess that the theme provider would do that.

Reply:
from the doc ```Add Notifications component anywhere in your application, usually it is placed inside [MantineProvider](https://mantine.dev/theming/mantine-provider/) next to your App component:```.  I will try to put it from the _app.tsx file and see how it goes

Reply:
Oh I didn't know the documentation was recommending this. If its mentioned in the docs, then it becomes more obvious that the theme provider does that. You can ignore my comment I guess.